tests/dataset_tests/parsers_tests/test_csv_file_parser.py

```diff
@@ -99,10 +113,14 @@ def test_csv_file_parser_target_index(csv_file, mols, label_a):
     """test `labels` option and retain_smiles=True."""
     preprocessor = NFPPreprocessor()
     parser = CSVFileParser(preprocessor, labels='labelA', smiles_col='smiles')
-    result = parser.parse(csv_file, return_smiles=True, target_index=[0, 2])
+    result = parser.parse(csv_file, return_smiles=True, target_index=[0, 2],
+                          return_is_successful=True)
     dataset = result['dataset']
     smiles = result['smiles']
     assert len(dataset) == 2
+    is_successful = result['is_successful']
+    assert numpy.alltrue(is_successful)
```

Comment:
Just a comment. If you would like to test `target` with `return_is_successful` option, using `csv_file_invalid` is better.